analytic_hr_department_restriction/migrations/18.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2025 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+from odoo.tools.sql import column_exists
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """Set the department_id value in the project company."""
+    if not column_exists(env.cr, "account_analytic_plan", "old_department_id"):
```

Comment:
Why not using openupgradelib method for doing the company dependent transformation?

Reply:
What would be the method to do this automatically? At https://github.com/OCA/helpdesk/pull/611 it was necessary to do this whole process to get the values right.